### PR TITLE
[UI] Add accessibility option to show all game tiles in color

### PR DESCRIPTION
**Pull request by connercsbn**
Patch to address feature request https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1301

This adds an option in accessibility settings that, when enabled, adds the `allTilesInColor` class to the GameCard component, making all tiles show in color. 

image
image

Just wanted to get this out there and get some feedback. Will do checklist soon!

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/screens/Accessibility/index.tsx [hunk 1]
```diff
@@ -162,6 +165,20 @@ export default function Accessibility() {
...
         <ThemeSelector />
+        <span className="setting">
+          <label className={classNames('toggleWrapper', { isRTL: isRTL })}>
+            <ToggleSwitch
+              value={allTilesInColor}
+              handleChange={() => {
+                setAllTilesInColor(!allTilesInColor)
+              }}
+              title={t(
+                'accessibility.allTilesInColor',
```
Comment (arielj) [inline src/screens/Accessibility/index.tsx:176]: Please use camel_case here `accessibility.all_titles_in_color`.

Also, please run `yarn i18next` after changing that so it updates all the locales for all languages.

**Comment by connercsbn** [comment]
Did something go wrong when I ran yarn i18next?

File: public/locales/fa/translation.json [hunk 2]
```diff
@@ -160,8 +181,20 @@
...
+        "saves": {
+            "syncing": ""
+        }
+    },
+    "launch": {
+        "options": "Launch Options..."
+    },
     "library": {
-        "gridView": "نمای شبکه‌ای",
+        "gridView": "نمای شبکه\u200cای",
```
Comment (imLinguin) [inline public/locales/fa/translation.json:197]: Could you remove this `\u200c` manually? this is i18n bug with languages like this

Comment (connercsbn) [reply public/locales/fa/translation.json]: 👍 https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1311/commits/3842ba3b58a7fcb42003ad4f47ffd56e356c8826

File: public/locales/bg/gamepage.json [hunk 3]
```diff
@@ -66,14 +64,6 @@
         "moving": "Преместване",
         "repairing": "Поправяне"
     },
-    "info": {
```
Comment (flavioislima) [inline public/locales/bg/gamepage.json:69]: those removals doesnt seems right

**Comment by flavioislima** [comment]
Need to check why so many translations were deleted on this PR. I cannot find any change related to that, so we need to check `main` and run the command there to see whats happening because it is deleting a lot of keys from settings, gamepage, etc.

File: public/locales/bg/gamepage.json (hunk 3 above)
Comment (connercsbn) [reply public/locales/bg/gamepage.json:69]: that's what I was thinking, but that's what happens when I run yarn i18next, even on the up-to-date main branch

Comment (flavioislima) [reply public/locales/bg/gamepage.json:69]: `yarn i18next` or `yarn i18n`? should be the same, I guess. But I will try to investigate this later.

**Comment by flavioislima** [comment]
@arielj I believe that the missing keys are related to one of the refactors on the UI.
Some keys moved to a different file and then lost the translation.
Example: `status.moving` was on the `gamepage.json` but moved to `translations.json` and lost the keys.
Maybe was when some components were moved. I dont know if this is breaking some language right now, but might be.
We should investigate that in another PR and try to move the keys around files, or maybe start using one single file for all translations.

**Comment by arielj** [comment]
Agree on using one single file for all the translations, since we don't really need to modify those files manually anyway, should simplify some code and allow re-usability

**Comment by flavioislima** [comment]
@arielj yes, that will be a lot of work, we have 64 translations file now, but will work best and it is better than losing all translations.

**Comment by flavioislima** [comment]
@connercsbn @arielj 
Found the issue and it is simple to solve.
Just edit this line:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/cb7b47560c1e244e38355fa3d757a64920e39573/src/screens/Game/GamePage/index.tsx#L48

to put it like this:
`const { t: t2 } = useTranslation()`

and then run `yarn i18n` again. In case it doesnt fix it, try removing the folder `locales` and then run the command again. Should be fine.

The thing is that for the default file, that is `translations.json` we dont need to pass an argument, because it is the default.
I mean, the i18next should be able to catch that but it doesnt.

File: public/locales/en/translation.json [hunk 4]
```diff
@@ -282,7 +286,6 @@
             "title": "Manual Sync Saves",
             "upload": "Upload"
         },
-        "maxRecentGames": "Played Recently to Show",
```
Comment (flavioislima) [inline public/locales/en/translation.json:285]: weird that it is still removing a few keys not related with the PR. 🤔

Comment (connercsbn) [reply public/locales/en/translation.json:285]: Ah, looks like it's caused by a couple deleted lines from here: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/commit/421ae61f5a1db85c83d6acac2f8601e515e9e452

Comment (flavioislima) [reply public/locales/en/translation.json:285]: Yes, you are right. Let me try to fix those

**Comment by flavioislima** [comment]
ok, fixed it in main. Try to update your branch now.
also tested the feature and it is working fine. :)

**Review by connercsbn — COMMENTED** [review]
Seems like last thing I need to do is revert the deletion of these lines, since they're being generated in English for these translation files. I'll see if I can grab the files from the main branch and rerun i18n

**Review by flavioislima — APPROVED** [review]
Perfect! Thanks a lot for that! :D

**Pull request merged**